docs/reference/search/request/highlighters-internal.asciidoc

```diff
@@ -140,11 +140,11 @@ Since the field `content` was not indexed either with offsets or term vectors,
 its raw field value will be analyzed, and in-memory index will be built from
 the terms that match the query:
 
-    {"token":"onli","start_offset":12,"end_offset":16,"position":3},
+    {"token":"only","start_offset":12,"end_offset":16,"position":3},
```

Comment:
The original tokens are correct here. The `english` analyzer stems `only` to `onli`.

You can verify this using the `_analyze` API:
```
GET /_analyze?pretty
{
  "analyzer" : "english",
  "text" : "only fox"
}
```

Response:

```
{
  "tokens" : [
    {
      "token" : "onli",
      "start_offset" : 0,
      "end_offset" : 4,
      "type" : "<ALPHANUM>",
      "position" : 0
    },
    {
      "token" : "fox",
      "start_offset" : 5,
      "end_offset" : 8,
      "type" : "<ALPHANUM>",
      "position" : 1
    }
  ]
}
```